FourSquare/Controllers/Base/PageBase/MenuItemViewController.swift

```diff
@@ -64,7 +62,6 @@ extension MenuItemViewController: UITableViewDataSource {
 
 extension MenuItemViewController: UITableViewDelegate {
     func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        let rowHeight: CGFloat = 140
         return rowHeight
```

Comment:
if all rows have the same row height, prefer use `rowHeight` property of `UITableView`